### PR TITLE
HDDS-2733. start/Kubernetes.md translated to Chinese

### PR DESCRIPTION
**Pull request by iamabug**
## What changes were proposed in this pull request?

one new .md file

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-2733

## How was this patch tested?

hugo server -D

### CODE REVIEW

File: hadoop-hdds/docs/content/start/Kubernetes.zh.md [hunk 1]
```diff
@@ -21,31 +21,32 @@ weight: 22
...
 cd kubernetes/examples/ozone
 kubectl apply -f .
 ```
 
-And you can check the results with
+用下面的命令检查结果：
 
 ```
 kubectl get pod
 Access the services
```
Comment (iamabug) [inline hadoop-hdds/docs/content/start/Kubernetes.zh.md]: `Access the services` does not makes sense to me here.

**Comment by iamabug** [comment]
@cxorm @xiaoyuyao Could you help review this, thanks.

File: hadoop-hdds/docs/content/start/Kubernetes.zh.md (hunk 1 above)
Comment (xiaoyuyao) [reply hadoop-hdds/docs/content/start/Kubernetes.zh.md]: I think it means access the ozone service via port forwarding.

File: hadoop-hdds/docs/content/start/Kubernetes.zh.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Ozone on Kubernetes
+title: 在 Kubernetes 上部署 Ozone
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/start/Kubernetes.zh.md]: Kubernetes=> Kubernetes集群

File: hadoop-hdds/docs/content/start/Kubernetes.zh.md [hunk 3]
```diff
@@ -21,31 +21,32 @@ weight: 22
...
 
-As the _apache/ozone_ docker images are available from the dockerhub the deployment process is very similar to Minikube deployment. The only big difference is that we have dedicated set of k8s files for hosted clusters (for example we can use one datanode per host)
-Deploy to kubernetes
+由于 _apache/ozone_ 镜像可以从 Docker Hub 获取到，K8s 上的部署过程和 Minikube 上的部署过程十分相似，唯一的区别是我们为 K8s 部署准备了专门的配置
+文件（比如，我们可以在每个 K8s 节点上部署一个 Datanode）。
 
-`kubernetes/examples` folder of the ozone distribution contains kubernetes deployment resource files for multiple use cases.
 
-To deploy to a hosted cluster use the ozone subdirectory:
+ozone 安装包中的 `kubernetes/examples` 目录包含了不同用途的部署资源文件。
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/start/Kubernetes.zh.md]: 包含了不同用途的部署资源文件=>包含了为不同用例设计的Kubernetes部署资源文件

File: hadoop-hdds/docs/content/start/Kubernetes.zh.md [hunk 4]
```diff
@@ -21,31 +21,32 @@ weight: 22
...
-And you can check the results with
+用下面的命令检查结果：
 
 ```
 kubectl get pod
 Access the services
 ```
 
-Now you can access any of the services. By default the services are not published but you can access them with port-foward rules.
+现在你可以访问 ozone 的各个服务，默认情况下它们并没有向外暴露，不过你可以通过设置端口转发规则进行访问：
```
Comment (xiaoyuyao) [inline hadoop-hdds/docs/content/start/Kubernetes.zh.md]: 向外暴露，不过你可以通过设置端口转发规则进行访问：=》向外开放ozone服务端口，不过你可以通过设置端口转发规则来开放外部访问。

File: hadoop-hdds/docs/content/start/Kubernetes.zh.md (hunk 1 above)
Comment (iamabug) [reply hadoop-hdds/docs/content/start/Kubernetes.zh.md]: So is translation necessary or not ?

Comment (xiaoyuyao) [reply hadoop-hdds/docs/content/start/Kubernetes.zh.md]: I think we can translate it to：访问ozone服务

**Pull request merged**